docs/contributing/sdk.md

```diff
@@ -0,0 +1,127 @@
+---
+id: sdk
+title: Contributing to XSOAR SDK
+slug: sdk
+authors: [kgal-pan]
+tags: [sdk,python,dev_env,contributing,pr]
+---
+
+# Contributing to XSOAR SDK
+
+Contributions to [`demisto-sdk`](https://github.com/demisto/demisto-sdk) are welcome and appreciated. 
+
+Some common ways to contribute are:
+
+- Create new commands.
+- Improve existing implementation.
+- Fix bugs.
+- Improve documentation. 
+
+### Prerequesites
+
+Make sure you already have the following requirements:
+
+* An active GitHub account.
+* A [fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) of the [`demisto-sdk`](https://github.com/demisto/demisto-sdk) repository. 
+* The forked repository cloned in your local machine or a codespace.
+* Have Python 3 and [`poetry`](https://python-poetry.org/docs/#installation) installed.
+* [Docker](https://docs.docker.com/engine/install/) installed.
+
+## How to Contribute
+
+Before contributing, we need to make sure we set up a development enviromment. Once that is ready, we can begin the development process which includes adding or modifying code, adding unit tests and documenting the changes. Upon completion, we can open a Pull Request to push those changes into the next release of the SDK. 
+
+### 1. Set Up Development Environment
+
+Run the following command from the repository root directory:
+
+1. Create a new branch to hold contributed work:
+
+```bash
+❯ git checkout -b $BRANCH_NAME
+```
+
+2. Install all required dependencies:
+
+```bash
+❯ poetry install
+```
+
+3. Activate the virtual environment:
+
+```bash
+❯ poetry shell 
+```
+
+Run the following command to see where the new virtual environment is saved in the filesystem:
+
+```bash
+❯ poetry env info
+
+Virtualenv
+Python:         3.10.8
+Implementation: CPython
+Path:           /path/to/pypoetry/virtualenvs/demisto-sdk-zRo7lI35-py3.10
+Executable:     /path/to/pypoetry/virtualenvs/demisto-sdk-zRo7lI35-py3.10/bin/python
+Valid:          True
+```
+
+This output will be helpful when selecting the Python interpreter in [Visual Studio Code](https://code.visualstudio.com/docs/python/environments) and [PyCharm](https://www.jetbrains.com/help/pycharm/configuring-python-interpreter.html#add-existing-interpreter).
+
+You should now have a working development environment. Feel free to jump into your favorite IDE and open the repository.
+
+### 2. Add Work
+
+This is the step where you add new features, fix bugs, etc., commit and push them.
```

Comment:
```suggestion
This is the step where you add new features, fix bugs, etc, commit and push them.
```

Reply:
etc. is always followed by a period, even mid-sentence, see https://www.grammarbook.com/blog/definitions/all-about-etc/#:~:text=The%20abbreviation%20etc.%20is%20from,in%20a%20period%2C%20even%20midsentence.